bciers/libs/utils/src/customTransformErrors.test.ts

```diff
@@ -42,6 +95,35 @@ const craBusinessNumberError = [
   },
 ];
 
+const latitudeError = [
+  {
+    name: "maximum",
+    property: ".section2.latitude_of_largest_emissions",
+    message: "must be <= 90",
+    params: {
+      comparison: "<=",
+      limit: 90,
+    },
+    stack: "'Latitude of Largest Point of Emissions' must be <= 90",
+    schemaPath:
+      "#/properties/section2/properties/latitude_of_largest_emissions/maximum",
+  },
+];
+const longitudeError = [
+  {
+    name: "minimum",
+    property: ".section2.longitude_of_largest_emissions",
+    message: "must be >= -180",
+    params: {
+      comparison: ">=",
+      limit: -180,
+    },
+    stack: "'Longitude of Largest Point of Emissions' must be >= -180",
+    schemaPath:
+      "#/properties/section2/properties/longitude_of_largest_emissions/minimum",
```

Comment:
it seems like the error messages here don't match the error messages expected in FacilityForm.test.tsx?